lib/msal-node/src/error/NodeAuthError.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AuthError } from "@azure/msal-common";
+
+/**
+ * NodeAuthErrorMessage class containing string constants used by error codes and messages.
+ */
+export const NodeAuthErrorMessage = {
+    hashEmptyError: {
```

Comment:
We never process `hash` in `msal-node`. This is done by the `user` passing us the auth code. I am curious why we need this error in `msal-node`